packages/wonder-blocks-tooltip/components/tooltip-portal-mounter.js

```diff
@@ -111,24 +102,31 @@ export default class TooltipPortalMounter extends React.Component<Props> {
         destination.parentNode.removeChild(destination);
     }
 
-    _timeoutDoMount() {
-        this._clearMountTimeout();
-        this._timeoutId = setTimeout(() => this._doMount(), 0);
-    }
+    _renderChildren() {
+        if (!this._destination) {
+            this._doMount();
+            return;
+        }
+
+        const {children} = this.props;
+        if (!children) {
+            this._doUnmount();
+            return;
+        } else {
+            ReactDOM.unmountComponentAtNode(this._destination);
```

Comment:
If we get here, the content of the bubble changed, so let's remove what we had and render something new.

Reply:
Not certain this is necessary, or if the later render call kinda handles this, but it seems appropriate and is certainly clearer

Reply:
These would be good comments to have in the code.